### PR TITLE
Support reload-on-change for local scripts

### PR DESCRIPTION
**Pull request by jherico**
Similar to work done in the Procedural.cpp file, detect if a script is a local file URL, if so, keep an eye on the modified time and reload the script if it changes.

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: 99d1e17ae8a08fd41c340cbd23ede360da0e117f available at: https://gist.github.com/ba6df46292f275f85dc4<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6021.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6021.dmg

File: libraries/script-engine/src/ScriptEngine.cpp [hunk 1]
```diff
@@ -1022,6 +1033,41 @@ void ScriptEngine::unloadAllEntityScripts() {
     _entityScripts.clear();
 }
 
+void ScriptEngine::refreshFileScript(const EntityItemID& entityID) {
```
Comment (ZappoMan) [inline libraries/script-engine/src/ScriptEngine.cpp]: are you calling unload()/preload() on these? it doesn't look like you are... that will break the paradigm

Comment (ZappoMan) [inline libraries/script-engine/src/ScriptEngine.cpp]: I see unload() gets called from unloadEntityScript() - so that's good.

**Comment by ZappoMan** [comment]
this doesn't seem to work on Windows 8.1... I've set an entity script to reference a local file, and I'm editing and saving the file, and it's not running the unload/preload methods

**Pull request merged**